js/src/utils.ts

```diff
@@ -310,3 +311,25 @@ export const getTokenizedDomains = async (
     })
     .filter((e) => !!e.reverse);
 };
+
+/**
+ * This function can be used to retrieve the domain price from its name
```

Comment:
A bit more detail here could be helpful I think. Maybe something like "This function can be used to retrieve the registration cost of a domain from its name."